crates/iota-light-client/README.md

```diff
@@ -20,45 +20,62 @@ The light client requires a config file and a directory to cache checkpoints, an
 
 ## Setup
 
-The config file for the light client takes a URL for a full node, a directory (that must exist) and within the directory to name of the genesis blob for the IOTA network.
+The config file for the light client takes a URL for a full node, a directory to store checkpoint summaries (that must exist) and within the directory the name of the genesis blob for the IOTA network.
 
 ```
-full_node_url: "https://api.testnet.iota.cafe:443"
-checkpoint_summary_dir: "checkpoints_dir"
-genesis_filename: "genesis.blob"
+rpc_url: "https://api.testnet.iota.cafe"
+graphql_url: "https://graphql.testnet.iota.cafe"
+checkpoints_dir: "checkpoints_testnet"
+# If not provided, make sure genesis.blob exists in checkpoints_dir
+genesis_blob_download_url: "https://dbfiles.testnet.iota.cafe/genesis.blob"
+sync_before_check: false
+object_store_url: "https://checkpoints.testnet.iota.cafe"
+archive_store_config:
+  object-store: "S3"
+  aws-endpoint: "https://archive.testnet.iota.cafe"
+  aws-virtual-hosted-style-request: true
+  no-sign-request: true
+  aws-region: "weur"
+  object-store-connection-limit: 20
+
 ```
 
-The genesis blob for the IOTA mainnet can be found here: https://github.com/iotaledger/iota-genesis/blob/main/mainnet/genesis.blob
+The genesis blob for the IOTA testnet can be found here: https://dbfiles.testnet.iota.cafe/genesis.blob. Download and place it inside the checkpoint directory.
```

Comment:
Can this be removed now as it's in the config and will be downloaded automatically?